src/sdg/api/serializers/core.py

```diff
@@ -9,7 +9,12 @@ class ProductenCatalogusSerializer(serializers.HyperlinkedModelSerializer):
     """Serializer for ProductenCatalogus details, including organizations and products."""
 
     organisatie = LokaleOverheidBaseSerializer(source="lokale_overheid")
-    producten = ProductBaseSerializer(many=True)
+    producten = ProductBaseSerializer(
+        many=True,
+        help_text="""Het product waar dit product van af hankelijk is dit geven we aan met een van de volgende velden:
```

Comment:
"Het product waar dit product van afhankelijk is. Dit geven we aan met een van de volgende velden"